modules/consentManagement.js

```diff
@@ -379,6 +379,13 @@ function storeConsentData(cmpConsentObject) {
       vendorData: (cmpConsentObject) ? cmpConsentObject.getVendorConsents : undefined,
       gdprApplies: (cmpConsentObject) ? cmpConsentObject.getConsentData.gdprApplies : gdprScope
     };
+  } else if (cmpConsentObject.addtlConsent && utils.isStr(cmpConsentObject.addtlConsent)) {
```

Comment:
couldn't this whole `else if` block be replaced with a separate `if` just before the `consentData.apiVersion = cmpVersion;` line that looks like this?

```
if (cmpConsentObject.addtlConsent && utils.isStr(cmpConsentObject.addtlConsent)) {
  consentData.addtlConsent = cmpConsentObject.addtlConsent;
}
```

Reply:
probably, although that would slightly change the functionality, looking for ac strings on v1 cmps.

Reply:
i made a change that preserved the functionality and removed the duplicate conditionals block, with a nested if.